src/MahApps.Metro/Controls/ColorPicker/ColorHelper.cs

```diff
@@ -24,28 +25,76 @@ public class ColorHelper
+        /// Gets a static default instance of <see cref="ColorHelper"/> with the original WPF color names.
+        /// </summary>
+        public static readonly ColorHelper DefaultInstanceInvariant = new(CultureInfo.InvariantCulture);
+
+        
+        /// <summary>
+        /// Creates a new Instance of the ColorHelper with the default translations.
+        /// </summary>
+        public ColorHelper() : this(CultureInfo.CurrentUICulture, typeof(ColorNames))
         {
-            ColorNamesDictionary = new Dictionary<Color, string>();
+            // Empty
+        }
 
-            var rm = new ResourceManager(typeof(ColorNames));
-            var resourceSet = rm.GetResourceSet(CultureInfo.CurrentUICulture, true, true);
+        /// <summary>
+        /// Creates a new Instance of the ColorHelper
+        /// </summary>
+        /// <param name="culture">The <see cref="CultureInfo"/> used to get the local color names</param>
+        public ColorHelper(CultureInfo? culture) : this(culture, typeof(ColorNames))
+        {
+            // Empty
+        }
 
-            if (resourceSet != null)
+        /// <summary>
+        /// Creates a new Instance of the ColorHelper
+        /// </summary>
+        /// <param name="culture">The <see cref="CultureInfo"/> used to get the local color names</param>
+        /// <param name="resourceDictionaryType">A type from which the <see cref="ResourceManager"/> derives all information for finding .resources files.</param>
+        public ColorHelper(CultureInfo? culture, Type resourceDictionaryType)
+        {
+            ColorNamesDictionary = new Dictionary<Color, string>();
+
+            if (culture is null)
             {
-                foreach (var entry in resourceSet.OfType<DictionaryEntry>())
+                foreach (var propertyInfo in typeof(Colors).GetProperties(BindingFlags.Static | BindingFlags.Public))
                 {
+                    var color = (Color)(propertyInfo.GetValue(null) ?? default(Color));
                     try
                     {
-                        if (ColorConverter.ConvertFromString(entry.Key.ToString()) is Color color)
+                        if (!ColorNamesDictionary.ContainsKey(color))
                         {
-                            ColorNamesDictionary.Add(color, entry.Value!.ToString()!);
+                            ColorNamesDictionary.Add(color, propertyInfo.Name);
                         }
                     }
                     catch (Exception)
                     {
-                        Trace.TraceError($"{entry.Key} is not a valid color key!");
+                        Trace.TraceError($"{color} could not be added to dictionary!");
+                    }
+                }
+            }
+            else
+            {
+                var rm = new ResourceManager(resourceDictionaryType);
+                var resourceSet = rm.GetResourceSet(culture, true, true);
+
+                if (resourceSet != null)
```

Comment:
Is it a better idea to do the same thing as above (fill with default values) instead of keeping an empty `ColorNamesDictionary`?